Dynatrace/dynatrace-operator
# Add an validator to verify correctness of ProxyUrl string/secret

Description
ActiveGate Proxy URL has to be properly [encoded](https://en.wikipedia.org/wiki/Percent-encoding) URI. ActiveGate POD fails if Proxy URL contains special characters (for instance "?) unencoded. To improve user experience I propose to add an Validator to reject a CR if the Proxy value or Proxy secret is invalid. 

How can this be tested?
Use any unencoded special character(s) in the Proxy Url (proxy.value, proxy secret) and try to apply your CR - should be rejected.

## Checklist
